r/R/feather.R

```diff
@@ -46,9 +48,14 @@
 #' @seealso [RecordBatchWriter] for lower-level access to writing Arrow IPC data.
 #' @seealso [Schema] for information about schemas and metadata handling.
 #' @examples
-#' tf <- tempfile()
-#' on.exit(unlink(tf))
-#' write_feather(mtcars, tf)
+#' tf1 <- tempfile(fileext = ".feather")
+#' tf2 <- tempfile(fileext = ".arrow")
+#' on.exit({
+#'   unlink(tf1)
+#'   unlink(tf2)
+#' })
+#' write_feather(mtcars, tf1, version = 1)
+#' write_ipc_file(mtcars, tf2)
```

Comment:
Would it make sense to consolidate the docs for read/write_ipc_stream with these now? I could see arguments both ways. (Side note: there's some stale documentation in the ipc_stream functions that should get cleaned up.)

Reply:
I think it would be easier to understand if the pages were separated for each type of file to be read/write, as they are in their current state.